megazords/ios/MozillaAppServices.xcodeproj/project.pbxproj

```diff
@@ -3,7 +3,7 @@
 	archiveVersion = 1;
 	classes = {
 	};
-	objectVersion = 52;
+	objectVersion = 51;
```

Comment:
I know it's XCode doing this but feels a little weird we're going down a version rather than up

Reply:
Yeah, it's weird, I think I will just manually revert this change